src/components/FilterChipsCarousel.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react'
+import { ScrollView, StyleProp, StyleSheet, Text, View, ViewStyle } from 'react-native'
+import Touchable from 'src/components/Touchable'
+import colors from 'src/styles/colors'
+import { typeScale } from 'src/styles/fonts'
+import { Spacing } from 'src/styles/styles'
+
+export interface FilterChip<T> {
+  id: string
+  name: string
+  filterFn: (t: T) => boolean
+}
+
+interface Props<T> {
+  chips: FilterChip<T>[]
+  selectedChips: FilterChip<T>[]
```

Comment:
Could we use `selectedChipIds` instead? ensures there's a single source of truth for these.